leap/README.md

```diff
@@ -0,0 +1,40 @@
+## leap: meta-gradient path learner in MXNet
+
+--------------------------------------------------------------------------------
+
+MXNet implementation of  "leap", the meta-gradient path learner published in ICLR 2019: [(link)](https://arxiv.org/abs/1812.01054) by S. Flennerhag, P. G. Moreno, N. Lawrence, A. Damianou.
+
+
+## Getting Started
+Check the `demos` and `test` folders. 
+
+
+## Installation
+* __Dependencies:__
+Primary dependencies are MXNet >=1.2. See all requirements in [setup.py](setup.py).
+* __Supported architectures / versions:__
+Python 3.6+ on MacOS and Amazon Linux. 
+
+
+-  __Install from source:__
+To install leap from source, after cloning the repository run the following from the top-level directory:
+```
+pip install .
```

Comment:
-e flag for editable for development. Potentially add separate instructions for development